software/builder/visionBuilder

```diff
@@ -134,36 +134,55 @@ function doLinkSrc() {
 
 # Get make options.
 function getMakeOpts() {
-  [[ "$CPUCOUNT" != "" ]] && return;
-  
+  MAKEOPTS=""
+
+  # Set our expectations regarding 'make' on this platform...
   local OS=`uname`
 
-  # Find our CPU count, default to 1.
-  CPUCOUNT=1
   case $OS in
   Linux)
+    [[ ! -v CPUCOUNT ]] && CPUCOUNT=$(grep -c processor /proc/cpuinfo)
```

Comment:
Just out of curiosity, what version of bash are you running?  I've been trying to anticipate what might work for you by using Centos 7, which supposedly tracks RHEL 7, as my reference OS.

Thanks,
Mike